lib/Dialect/LWE/IR/NewLWEAttributes.td

```diff
@@ -314,19 +311,55 @@ def LWE_CiphertextSpaceAttr : AttrDef<LWE_Dialect, "CiphertextSpace"> {
     An attribute describing the ciphertext space and the transformation from
     plaintext space to ciphertext space of an FHE scheme.
 
-    The ciphertext space information includes the ring structure, which contains
-    the ciphertext modulus $q$. Ciphertexts using an RNS representation for $q$
-    will represent their ciphertext components in the ring attribute. Scalar LWE
-    ciphertexts (as opposed to RLWE) use an ideal polynomial of degree 1, $x$.
-    CGGI ciphertexts will typically use a power of two modulus.
+    The ciphertext space information includes the ring attribute, describing the
+    space that the ciphertext elements belong to. The ring attribute contains a
+    coefficient type attribute that describes the semantics of the coefficient.
+    For example, a ring modulo $1 + x^1024$ with coefficients modulo $q =
+    298374$ will be described as
+
+    ```
+    !ideal = !polynomial.int_polynomial<1 + x**1024>
+    !cmod = !modarith<storage=i64, modulus=298374>
```

Comment:
Maybe slap a TODO here to double-check this syntax against the syntax that we ultimately use for the new modarith and rns types. At least, it will need a dialect prefix before the type name.

Reply:
Nice point I'll synthesize the poly changes into a tracking issue.